packages/playground/src/components/pull-request/sections/pull-request-merge-section.tsx

```diff
@@ -0,0 +1,123 @@
+  CheckCircleSolid,
+  Clock,
+  ChatBubble
+} from '@harnessio/icons-noir'
+import { CheckStatus, TypeCheckData } from '../interfaces'
+import { timeDistance } from '../utils'
+
+interface PullRequestMergeSectionProps {
+  checkData: TypeCheckData[]
+  checksInfo: { header: string; content: string; status: CheckStatus }
+}
+const PullRequestMergeSection = ({ checkData, checksInfo }: PullRequestMergeSectionProps) => {
+  const [isExpanded, setExpanded] = useState(false)
+
+  return (
+    !isEmpty(checkData) && (
+      <div className=" pt-4 pb-3  border-b">
+        <div className="flex justify-between">
+          <div className="flex ">
+            {checksInfo.status === CheckStatus.PENDING ? (
+              <Clock className=" mt-1 " />
+            ) : checksInfo.status === CheckStatus.FAILURE || checksInfo.status === CheckStatus.ERROR ? (
+              <WarningTriangleSolid className="text-destructive mt-1 " />
+            ) : checksInfo.status === CheckStatus.RUNNING ? (
+              <ChatBubble className="text-warning mt-1 " />
+            ) : (
+              <CheckCircleSolid className="text-success mt-1 " />
+            )}{' '}
+            <div className="pl-4 flex flex-col">
+              <Text size={2}>{checksInfo.header}</Text>
+              <Text className="text-tertiary-background" size={1}>
+                {checksInfo.content}
+              </Text>
+            </div>
+          </div>
+          <button
+            // className={cx('mr-3')}
+            onClick={() => {
+              setExpanded(!isExpanded)
+            }}>
+            {isExpanded ? 'Show less' : 'Show more'}
+            {isExpanded ? <NavArrowUp className="pt-1" /> : <NavArrowDown className="pt-1" />}
+          </button>
+        </div>
+
+        {isExpanded ? (
+          <div className="flex flex-col pl-4 mx-4 pt-2">
+            {checkData.map(check => {
+              const time = timeDistance(check.check.created, check.check.updated)
+              return (
+                <div className="flex  justify-between py-2 border-t ">
+                  <div className="flex">
+                    {check.check.status === CheckStatus.PENDING ? (
+                      <Clock className=" mt-1 " />
+                    ) : check.check.status === CheckStatus.FAILURE || check.check.status === CheckStatus.ERROR ? (
+                      <WarningTriangleSolid className="text-destructive mt-1 " />
+                    ) : check.check.status === CheckStatus.RUNNING ? (
+                      <ChatBubble className="text-warning mt-1 " />
+                    ) : (
+                      <CheckCircleSolid className="text-success mt-1 " />
+                    )}
+
+                    <div className="truncate min-w-[200px] max-w-[200px] pl-3 pt-0.5 "> {check.check.identifier}</div>
+                    <div className="truncate  max-w-[200px] pl-3 pt-0.5 ">
+                      {' '}
+                      {check.check.status === CheckStatus.SUCCESS
```

Comment:
Can we move this to a `switch` statement to help with code readability?

Reply:
done